modules/core/src/main/java/org/apache/ignite/internal/processors/query/GridQueryProcessor.java

```diff
@@ -1283,8 +1287,11 @@ private void registerTypeLocally(String clsName) throws BinaryObjectException {
 
             Class<?> cls = U.box(U.classForName(clsName, null, true));
 
-            if (cls != null)
-                binProc.binaryContext().registerClass(cls, true, false, true);
+            if (cls != null) {
+                if (!platformOnly) {
```

Comment:
For single-line if() statement, curly braces are not needed.

Reply:
Fixed